Makefile

```diff
@@ -8,13 +8,13 @@ NAME = redis
 all: $(NAME) client server
 
 $(NAME):
-	$(CC) $(SRCS) $(CXXFLAGS) -o $(NAME)
+	$(CC) $(CXXFLAGS) $(SRCS) -o $(NAME)
 
 client:
 	g++ -Wall -Werror -Wextra -DASIO_STANDALONE -std=c++11 client.cpp -o client
```

Comment:
Why not just use the `$(CC) $(CXXFLAGS)` flags while compiling if they're already defined?